test/utils.js

```diff
@@ -167,8 +164,15 @@ var TestUtils = {
                     pt[k] = Math.round(v * 10000) / 10000;
                 }
             }
-        })
-        .value();
+        });
+        
+        if (massageOptions.sort) {
+            massageOptions.sort.forEach(function(sortField) {
+                arr = _.sortBy(sortField);
```

Comment:
This sets the value of `arr` to a sorted array of the characters in `sortField`...